frontend/src/views/GNewShoot.vue

```diff
@@ -491,6 +520,7 @@ export default {
       const enableStaticTokenKubeconfig = get(shootResource, 'spec.kubernetes.enableStaticTokenKubeconfig')
       const purpose = get(shootResource, 'spec.purpose')
       this.purpose = purpose
+      const workers = get(shootResource, 'spec.provider.workers')
```

Comment:
Here we use lodash `get(shootResource, 'spec.provider.workers')` somewhere else in this PR we use `shootItem?.spec?.provider?.workers `. What is our prefered way to access deep propeties?

Reply:
I usually only use lodash get in code where we already use it.